sdk/core/Azure.Core/src/Shared/NoBodyResponse{T}.cs

```diff
@@ -24,6 +24,11 @@ public override T Value
 
         public override Response GetRawResponse() => _response;
 
+        public override string ToString()
+        {
+            return $"Status: {GetRawResponse().Status}, Service returned no content";
```

Comment:
Can you add similar to DelayedResponse in batching?  I think the base ToString might cause it to throw.

Reply:
DOne